docs/readmes/orc8r/deploy_build.md

```diff
@@ -76,32 +83,31 @@ We'll build the Orchestrator Helm charts, as well as publish them to a
 To start, create a private GitHub repo to use as your Helm chart repo. We'll
 refer to this as `GITHUB_REPO`.
 
-Next, package the Magma Helm charts and publish them to the GitHub
-repo. You'll have to check out a temporary commit to build the `1.4.36` version
-of the orc8r chart for 1.3.x because `1.4.37` on the head of the release
-branch has some changes that you shouldn't try to deploy yet.
+Next, define some necessary variables
 
 ```bash
-cd MAGMA_ROOT
-git checkout a7580153
+export GITHUB_REPO=<github_repo>
```

Comment:
Nit for this code block: prefer e.g. `GITHUB_REPO=GITHUB_REPO_NAME` rather than `GIHTUB_REPO=<github_repo>` just because it formats nicer with the bash syntax